pkg/tuf_v2/repository/map.json

```diff
@@ -0,0 +1,13 @@
+{
+    "repositories": {
+      "public-good-instance": ["https://storage.googleapis.com/sigstore-tuf-root/"]
+    },
+    "mapping": [
+      {
+        "paths":        ["*"],
+        "repositories": ["Public-Good-Instance"],
```

Comment:
Is the naming capitalization different between L3 intentional? Looking at the tap4 spec, I'd expect this to be: "public-good-instance".